control/init.sh

```diff
@@ -0,0 +1,61 @@
+#! /bin/bash
```

Comment:
This script initializes a developer account for ansible deploys. There is no longer a single shared ansible repo. Instead, each developer has a clone from which to initiate deploys. Should help with people stepping on other people's work. The control repo is symlinked in from the `/home/ansible` (more on that below).